content/blog/2016/2016-09-22-jenkins-world-2016-wrap-up-pipeline.adoc

```diff
@@ -0,0 +1,141 @@
+---
+layout: post
+title: "Jenkins World 2016 Wrap-up - Pipeline"
+tags:
+- event
+- jenkinsworld
+author: lnewman
+---
+
+NOTE: This is a guest post by link:https://github.com/bitwiseman[Liam Newman],
+Technical Evangelist at Cloudbees.
+
+image:/images/conferences/Jenkins-World_125x125.png[Jenkins World 2016, role=right]
+
+I think link:/doc/pipeline[Jenkins Pipeline] is cool.
```

Comment:
This first sentence seems out of place, I think you could drop it and make the "As someone who" sentence the opener